front/app/containers/Authentication/Modal.tsx

```diff
@@ -84,6 +87,20 @@ const HEADER_MESSAGES: Record<Step, MessageDescriptor | null> = {
   'verification-success': null,
 };
 
+const getHeaderMessage = (
+  step: Step,
+  action:
+    | 'visiting'
+    | IInitiativeAction
+    | IParticipationContextPermissionAction
+    | IFollowingAction
+) => {
+  if (action === 'following') {
+    return messages.beforeYouFollow;
```

Comment:
Does this mean for every authentication modal step the header will have "Before you follow" rather than the other existing copy (e.g. Won't have "Confirm your email" when it gets to the email confirmation step)?